drugbank.py

```diff
@@ -202,5 +204,8 @@ def get_pubchem_cid(sid):
     target_id = "Uniprot:" + uniprot_id if uniprot_id else "DrugBank:" + be_id
 
     # TODO: Generate as directional (ListLink) for all of them for now
-    evalink(action, "MoleculeNode", "MoleculeNode", standard_id, target_id)
-    evalink("has_name", "MoleculeNode", "ConceptNode", target_id, name)
+    target_id = find_mol_type(target_id)
```

Comment:
This `target_id` here could actually be a Uniprot, but `find_mol_type` doesn't check for uniprot, so maybe we can either combine this with line 204, or do a similar if-else condition for this line?

Reply:
Thanks. I just added a condition to check for Uniprots.